styles/message.mcss

```diff
@@ -81,12 +85,12 @@ Message {
       a.avatar {
         img {
           width: 50px
+          height: 50px
```

Comment:
I'd go with fixing the `height` in that case, as the design is more sensitive to the vertical requirement of 50px... the width can be whatever mad aspect ratio the user wants... (he says, immediately regretting it...)